maptivist/src/main/webapp/script.js

```diff
@@ -181,79 +110,74 @@ function inBoundary(marker){
   var bounds = map.getBounds().toJSON();
   return (marker.latitude <= bounds.north && 
          marker.latitude >= bounds.south &&
-         marker.longitude <= bounds.west &&
-         marker.longitude >= bounds.east);
+         marker.longitude >= bounds.west &&
+         marker.longitude <= bounds.east);
 }
 
 /** Adds a new Marker based on id array
  */
-function drawMarker({id, latitude, longitude, title, description, links, categories}) {
-  // Get the coordinates from the marker class and create a new Latlng object
-  var myLatlng = new google.maps.LatLng(latitude, longitude);
-  
+function drawMarker(markerObj) {
+  // Get attributes from the marker class and create a new Latlng object
+  var title = markerObj.title;
+  var myLatlng = new google.maps.LatLng(markerObj.latitude, markerObj.longitude);
+
   // Create a new marker, it assumed that the position is a private attribute that cannot be accessed
   var marker = new google.maps.Marker({
     position: myLatlng,
     map: map,
     title: title
   });
 
-  // Adds the new marker to the map and pans to the marker 
+  // Adds the new marker to the map 
   marker.setMap(map);
-  map.panTo(marker.getPosition());
 
   // Adds the new infowindow to the marker
-  var infowindow = drawInfowindow(id, title, description, links, categories, marker.getPosition());
-  marker.addListener('click', function() {
+  var infowindow = drawInfowindow(markerObj);
+  marker.addListener('click', ()  => {
     infowindow.open(map, marker);
   });
-
-  // Adds the marker's random, unique id to list of already created markers
-  drawn_markers.push(id);
 }
 
-
-function drawInfowindow(id, title, description, links, categories, position) {
-  //Set info window content from Marker class
-
-  var location = position.toString();
-  
+/**
+* Adds an infowindow based on the marker creation form
```

Comment:
Add what "markerObj" is in the documentation here. (a json object).

Reply:
It's a single marker object that is a part of the JSON array